sys/include/test_utils/result_output.h

```diff
@@ -21,6 +21,9 @@
  * adapted. There can also be python layers that coordinate the output results,
  * ideally done with riotctrl.
  *
+ * Only one implementation should be selected, for example,
+ * test_utils_result_output_json.
```

Comment:
Wasn't there original a requirement to be able to select the output format at runtime? Maybe I am misremembering.

Reply:
That was what @kaspar wanted but I think the overhead would be too much. If the CI uses an implementation that can be easily reproduced by a developer then I think those concerns should be address.

Reply:
So in the future CI is building different code paths than a local build, by default?

Reply:
Nope, not now that json is default.

Reply:
ok!